packages/transition-common/src/services/accessibilityMap/TransitAccessibilityMapRouting.ts

```diff
@@ -157,11 +157,6 @@ class TransitAccessibilityMapRouting extends ObjectWithHistory<AccessibilityMapA
             this.errors.push(...queryAttrErrors);
         }
 
-        if (!attributes.scenarioId) {
```

Comment:
Why remove this?

Reply:
La vérification du scenarioId se fait déjà dans la fonction validateTrQueryAttributes qui est appelé juste avant, donc on recevait l'erreur "transit:transitRouting:errors:ScenarioIsMissing" en double.